intake_bluesky/mongo_embedded.py

```diff
@@ -118,14 +118,14 @@ def get_header_field(field):
                         if field[0:6] == 'count_':
                             return 0
                         else:
-                            return None
+                            return {}
 
                 def get_resource(uid):
                     resources = get_header_field('resources')
                     for resource in resources:
                         if resource['uid'] == uid:
                             return resource
-                    return None
+                    return {}
```

Comment:
This should either return a valid resource document or raise an error, probably a `KeyError`. Same for `get_datum`.

Reply:
It's checking what mongo_normalized does in this situation. Not sure if I covered it.

Reply:
Those raise ValueErrors in normalized.

Reply:
Updated